pkgs/top-level/all-packages.nix

```diff
@@ -14319,7 +14319,7 @@ with pkgs;
     inherit (python3Packages) filecheck;
   };
 
-  bluespec = callPackage ../development/compilers/bluespec {
+  bluespec = callPackage ../by-name/bl/bluespec/package.nix {
     gmp-static = gmp.override { withStatic = true; };
   };
```

Comment:
Is this override even still needed? Maybe the build system sets `-static`, but I can't remember.

Reply:
I don't know either and I also don't have the time to dig into it. Could we make this a separate pr?

Reply:
I just checked -- the override isn't needed, but we can make that a separate PR too.

Reply:
@jcumming Could you please share what you have found?